switchbot_ros/msg/StripLight.msg

```diff
@@ -0,0 +1,6 @@
+Header  header       # timestamp
+
+string  power        # ON/OFF state
+string  color        # the color value, RGB "255:255:255"
```

Comment:
How about split this field into `color_r`, `color_g`, `color_b` integers instead of string so that we can easily set value to here. There are some languages that do not support rich string manipulation.

Reply:
I split `color` int `color_r`, `color_g` and `color_b`.

``` python
                        rgb_string       = status['color']
                        r, g, b = map(int, rgb_string.split(':'))
                        msg.color_r      = r
                        msg.color_g      = g
                        msg.color_b      = b
```